specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-06-15/bms.json

```diff
@@ -1842,11 +1816,6 @@
             "Passed",
             "ActionRequired",
             "ActionSuggested",
-            "Healthy",
```

Comment:
remove enum value is kind of breaking change.  this change is really required ?

Reply:
Hi @akning-ms  these enum values were not populated in the service code. So it will not be a breaking change.

Reply:
agree with @akning-ms , removing an enum value is breaking change in SDK.